app/pulp/app/settings.py

```diff
@@ -113,9 +112,11 @@
     'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',),
     'DEFAULT_PAGINATION_CLASS': 'pulp.app.pagination.UUIDPagination',
     'PAGE_SIZE': 100,
-    'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAuthenticated',),
 }
 
+if not DEBUG:
+    REST_FRAMEWORK['DEFAULT_PERMISSIONS_CLASSES'] = ('rest_framework.permissions.IsAuthenticated',)
```

Comment:
I'll at least add a comment, and include it in the wiki page for "stuff I learned while working on this that's useful but not useful enough to put in the dev guide quite yet".